shell/components/form/ArrayList.vue

```diff
@@ -107,6 +111,9 @@ export default {
     showAdd() {
       return this.addAllowed;
     },
+    disableAdd() {
+      return this.addDisabled;
+    },
```

Comment:
NOTE: I think it's a little redundant to have a computed prop. Without it, the template would be just as simple and declarative. We already employ this pattern for `showAdd` and `showRemove`, so it makes sense to stick with the prevalent pattern.